AdyenActions/Components/3DS2/ThreeDS2Component.swift

```diff
@@ -92,9 +106,12 @@ public final class ThreeDS2Component: ActionComponent {
     /// - Parameter context: The context object for this component.
     /// - Parameter configuration: The component's configuration.
     public init(context: AdyenContext,
-                configuration: Configuration = Configuration()) {
+                configuration: Configuration = Configuration(),
+                presentationDelegate: PresentationDelegate?) {
```

Comment:
parameter is missing in the function documentation

Reply:
Done. Thanks ✅